libsolidity/formal/CHC.cpp

```diff
@@ -994,10 +986,11 @@ void CHC::resetSourceAnalysis()
 	ArraySlicePredicate::reset();
 	m_blockCounter = 0;
 
-	bool usesZ3 = false;
+	/// At this point every enabled solver is available.
+	/// If more than one Horn solver is selected we go with z3.
```

Comment:
Why? This sounds like something that should be documented somewhere to make it clear to the user. E.g. in the CLI option description.

Or maybe we should at least issue a warning in this case?